AnnoyancesFilter/sections/mobile-app.txt

```diff
@@ -512,12 +512,17 @@ cna.com.tw##.qrcode_box
 ||static.fastpic.ru/android.png
 itao.com###itao-bottom
 stitcher.com###appnag
+9gag.com##a[class="get-the-app-banner"]
+||9gag.com/s/*/static/dist/*/web/img/sidebar-download-app.png
 outlook.live.com##.notificationBarSlideIn
 yadi.sk##.mobile-badges
 maximonline.ru##.apps
 sahibinden.com##.download-app-banner
 ||dumskaya.net/js/appbanner/
 bing.com##.mt-edge-extension
+duckduckgo.com##div[class^="badge-link"]
+duckduckgo.com##div[class^="content-info"]
+duckduckgo.com##div[class$="ddg-extension-hide"]
```

Comment:
In general ok, but `^` are not required.
Added https://github.com/AdguardTeam/AdguardFilters/commit/1d3097e2d77bf6bab9f7e334b2d304b16db8b3eb